spec/user_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'user'
+require 'database_helpers'
+
+describe User do
+
+  describe '.create' do
+    it 'creates a new user' do
+      user = User.create(name: 'Test Person', handle: '@mrtest', email: 'test@example.com', password: 'password123')
```

Comment:
I like that your users have names and handles so that you aren't displaying their email address publicly on the peep feed.